content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1459,14 +1459,14 @@ Defines a multi-field ID (composite ID) on the model.
-| `name`      | **No**   | `String`           | The name that the Client API will expose for the argument covering all fields, e.g. `fullName` in `fullName: { firstName: "First", lastName: "Last"}`                                                                                         |
-| `map`       | **No**   | `String`           | The name of the underlying primary key constraint in the database                                                                                                                                                                             |
-| `length`    | **No**   | `number`           | Allows you to specify a maximum length for the subpart of the value to be indexed.<br /><br />MySQL only. In preview in versions 3.5.0 and later, and in general availability in versions 4.0.0 and later.                                    |
-| `sort`      | **No**   | `String`           | Allows you to specify in what order the entries of the ID are stored in the database. The available options are `Asc` and `Desc`.<br /><br />In preview in versions 3.5.0 and later, and in general availability in versions 4.0.0 and later. |
-| `clustered` | **No**   | `Boolean`          | Defines whether the ID is clustered or non-clustered. Defaults to `true`.<br /><br />SQL Server only. In preview in versions 3.13.0 and later, and in general availability in versions 4.0.0 and later.                                       |     |
+| Name        | Required | Type               | Description                                                                                                                                                                                                                                                    |
+| ----------- | -------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `fields`    | **Yes**  | `FieldReference[]` | A list of field names - for example, `["firstname", "lastname"]`                                                                                                                                                                                               |
+| `name`      | **No**   | `String`           | The name that the Client API will expose for the argument covering all fields, e.g. `fullName` in `fullName: { firstName: "First", lastName: "Last"}`                                                                                                          |
+| `map`       | **No**   | `String`           | The name of the underlying primary key constraint in the database.<br /><br />Not supported for MySQL or MongoDB.                                                                                                                                              |
```

Comment:
Not necessary to list a `@@id` is not supported for MongoDB at all and that is mentioned above via an admonition:
```suggestion
| `map`       | **No**   | `String`           | The name of the underlying primary key constraint in the database.<br /><br />Not supported for MySQL.                                                                                                                                              |
```